graphjet-core/src/main/java/com/twitter/graphjet/algorithms/TweetIDMask.java

```diff
@@ -20,11 +20,7 @@
 /**
  * The bit mask used to manipulate tweet ids to encode further information.
  */
-public final class TweetIDMask {
-
-  // Utility class
-  private TweetIDMask() {
-  }
+public final class TweetIDMask implements IDMask {
```

Comment:
Mostly maintaining the existing behavior, but the main thing about declaring final classes is to show that they should not be extended, which makes sense in this case. You should only extend IDMask, but not extend the subclasses in any way.